### PR TITLE
Simple feature/session sample

### PR DESCRIPTION
**Pull request by soywiz**

### CODE REVIEW

**Comment by soywiz** [comment]
Added several configuration samples to handle storing the contents in the client and the server.
Added a proper README explaining how it works.

File: feature/sessions/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+```bash
+./gradlew sessions:run
+```
+
+This document has been written for Ktor 0.9.1
+
+## Transfer using Cookies or Custom Headers
+
+You can either use cookies or custom HTTP headers for sessions. The code would be roughly the same but you have to
```
Comment (B1ggDave) [inline feature/sessions/README.md]: The code will be roughly the same, the only difference is you have to...

File: feature/sessions/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+call `cookie` method, or `header` method depending where do you want to send session information.
```
Comment (B1ggDave) [inline feature/sessions/README.md]: depending on where you want to send the session information.

File: feature/sessions/README.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+### Cookies
+
+Cookie method is intended for browser sessions. It will use a standard
+[`Set-Cookie` header](https://developer.mozilla.org/es/docs/Web/HTTP/Headers/Set-Cookie).
+Inside cookie block, you have access to a `cookie` property that allows you to configure the `Set-Cookie` header
```
Comment (B1ggDave) [inline feature/sessions/README.md]: Inside the cookie block

**Comment by elizarov** [comment]
Merged

**Pull request merged**